monkey/infection_monkey/ransomware/__init__.py

```diff
@@ -0,0 +1,10 @@
+import logging
+
+from infection_monkey.config import WormConfiguration
+
+LOG = logging.getLogger(__name__)
+
+
+def start_ransomware():
+    LOG.info(f"Windows dir configured for encryption is {WormConfiguration.windows_dir_ransom}")
+    LOG.info(f"Linux dir configured for encryption is {WormConfiguration.linux_dir_ransom}")
```

Comment:
We should put this in a class.

Reply:
This is just a stub, we should remove it altogether, but that's the responsibility of other PR's, which will know the classname and what responsibilities those classes have.